homeassistant/components/frontier_silicon/media_player.py

```diff
@@ -60,6 +61,7 @@
         vol.Required(CONF_HOST): cv.string,
         vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
         vol.Optional(CONF_PASSWORD, default=DEFAULT_PASSWORD): cv.string,
+        vol.Optional(CONF_NAME, default=""): cv.string,
```

Comment:
Remove the default parameter.
```suggestion
        vol.Optional(CONF_NAME): cv.string,
```